en/SetupView/safety_ardupilot.md

```diff
@@ -0,0 +1,202 @@
+
+<!-- Question 
+- no radio failsafe?
+-->
+
+
+### Return to Launch {#rtl_plane}
+
+This panel sets the [RTL Mode](http://ardupilot.org/copter/docs/rtl-mode.html) behaviour.
+
+![Safety Setup - RTL (Plane)](../../assets/setup/safety/safety_arduplane_return_to_launch.jpg)
+
+The configuration options are:
+- Select RTL return altitude ([RTL_ALT](http://ardupilot.org/copter/docs/parameters.html#rtl-alt-rtl-altitude)):
+  - **Return at current altitude** - Return at current altitude.
+  - **Return at specified altitude** - Ascend to specified altitude to return if below current altitude.
+
+
+### Arming Checks {#arming_checks_plane}
+
+[Arming Checks](#arming_checks_copter) are the same as for copter.
+
+
+## Rover
+
+The Rover safety page is shown below. For more information see [Failsafes](http://ardupilot.org/rover/docs/rover-failsafes.html).
+
+![Safety Setup - Rover (Ardupilot)](../../assets/setup/safety/safety_ardurover.jpg)
+
+
+
+### Battery Failsafe {#battery_failsafe_rover}
+
+The rover battery failsafe is the same as for [copter](#battery_failsafe).
+
+
+### Failsafe Triggers {#failsafe_triggers_rover}
+
+This panel enables the rover [Failsafes](http://ardupilot.org/rover/docs/rover-failsafes.html).
+
+![Safety Setup - Failsafe Triggers (Rover)](../../assets/setup/safety/safety_ardurover_failsafe_triggers.jpg)
+
+The configuration options are:
+
+- **Ground Station failsafe** ([FS_GCS_ENABL](http://ardupilot.org/rover/docs/parameters.html#fs-gcs-enable-gcs-failsafe-enable)) - Check to enable GCS failsafe.
+- **Throttle failsafe** ([FS_THR_ENABLE](http://ardupilot.org/rover/docs/parameters.html#fs-thr-enable-throttle-failsafe-enable)) - Enable/disable throttle failsafe (value is *PWM threshold* below).
+- **PWM threshold** ([FS_THR_VALUE](http://ardupilot.org/rover/docs/parameters.html#fs-thr-value-throttle-failsafe-value)) - PWM value below which throttle failsafe triggers.
+- **Failsafe Crash Check** ([FS_CRASH_CHECK](http://ardupilot.org/rover/docs/parameters.html#fs-crash-check-crash-check-action)) - What to do in the event of a crash: Disabled, Hold, HoldAndDisarm
+
+
+### Arming Checks {#arming_checks_rover}
+
+[Arming Checks](#arming_checks_copter) are the same as for copter.
+
+
+## Sub
+
+The Sub safety page is shown below. For more information see [Failsafes](https://www.ardusub.com/operators-manual/failsafes.html).
+
+![Safety Setup - Sub (Ardupilot)](../../assets/setup/safety/safety_ardusub.jpg)
+
+
+### Failsafe Actions {#failsafe_actions_sub}
+
+The configuration options are:
+
+- **GCS Heartbeat** - Select one of: Disabled, Warn only, Disarm, Enter depth hold mode, Enter surface mode.
+- **Leak** - Select one of: Disabled, Warn only, Enter surface mode.
+  - **Detector Pin** - Select one of: Disabled, Pixhawk Aux (1-6), Pixhawk 3.3ADC(1-2), Pixhawk 6.6ADC.
+  - **Logic when Dry** - Select one of: Low, High.
+- **Battery** - ?.
```

Comment:
Battery selector empty in Sub. Is that OK?

Reply:
I don't know anything about Sub. You'll have to ask the Sub folks. @jaxxzer?

Reply:
I'll tune in today and take a look at this.

Reply:
@hamishwillee can you show me what you are seeing? Our failsafes (on stable firmware) are documented here: https://www.ardusub.com/operators-manual/failsafes.html

Reply:
@jaxxzer This is recent (in the last week) master, on Ubuntu 16.04 using QGC 3.4.3.

Battery field is empty, and dropdown is empty:
![image](https://user-images.githubusercontent.com/5368500/46180114-e0480500-c301-11e8-9de7-d935a2dccf48.png)

Might be related to
![image](https://user-images.githubusercontent.com/5368500/46180068-9a8b3c80-c301-11e8-9bee-f68193ebe5ea.png)

Reply:
Ok. It appears that you are on v3.6.0-dev of ArduSub, which qgc does not support at the moment. We are working on bringing things up to speed for ArduSub on QGC master here.

Reply:
Thanks!